hack/dev-functions.sh

```diff
@@ -43,6 +43,18 @@ function register_argocd_cluster() {
 	kubectl config use-context "$prevcontext"
 }
 
+function label_argocd_cluster() {
```

Comment:
Sorry if it is a silly question, but how does the labels fit with the debug configuration? I looked at the other changes, but couldn't see the connection between them 😞

Reply:
The labeling affects how the progressive sync controller is going to apply https://github.com/Skyscanner/applicationset-progressive-sync/pull/97/files#diff-adb910ba444459702e453de3f6ed05206831fe068d92bc236e44e1bfe6303a81R1

So when as part of `make debug` we deploy the test prog sync object, we will sort of end up having a stable repeatable env for local testing/debugging.